db/seeds/qa.rb

```diff
@@ -15,56 +15,58 @@
   ContactType.create(contact_type_group: group, name: "Supervisor")
 end
 
+#
+# Config
+#
 # number of volunteer users and casa cases to generate
-VOLUNTEER_USER_COUNT = 100
-CASA_CASE_COUNT = 150
-SUPERVISOR_COUNT = 5
-
-SEED_PASSWORD = "123456"
+volunteer_user_count = 100
+casa_case_count = 150
+supervisor_count = 5
+seed_password = "123456"
+casa_admin_count = 3
```

Comment:
I think this is a good change to get rid of the constants.

Reply:
Want me to refactor it a little bit? I see some areas it could be improved.

Reply:
I’ve been thinking about this and wondering if we can use env variables and just have one seed file.

Reply:
We could. Given the environment we can set some options to add certain data. What I like of the current implementation is that each file has it's environment responsibility, so I kinda have mixed feelings there.

Reply:
Yeah, but if we need to test and are duplicating logic (I think the files are the same other than the variables), it seems worth it to just have one.

Reply:
That's a good idea to store the counts as environment variables. I think the only reason there are separate files is that we wanted a lot of users created on staging (and qa?), but not that many on development.

Reply:
Yeah, we should still be able to have the differences without having separate files. Unless there’s another difference I’m missing 😬

Reply:
I can work on this if you want. This PR would remain the same since we're checking to not raise exceptions.

Reply:
I think one file would be better as it seems that the staging and qa seed files have not been getting updated regularly like the development file has, and it seems that the only difference is the quantity of records that are being created.

Reply:
Can this PR be merge then? Let's create an issue for the seed refactor

Reply:
I opened a new issue. If you want to take it, comment and feel free, otherwise I’ll probably try to do it Monday morning https://github.com/rubyforgood/casa/issues/1035

Reply:
Confirmed, we want lots of data on staging but not lots of data on "dev" (local) environments, because we want realism in staging (because we do user acceptance testing and user training in staging) but we want not-slow-to-load in local dev environments, especially for contributors using less fancy hardware. 
As for QA, it didn't exist when we made this decision and I don't have a preference. :)